cogboard-webapp/src/components/widgets/dialogFields/TextInput.js

```diff
@@ -9,6 +9,7 @@ const TextInput = props => {
         shrink: true,
       }}
       margin="normal"
+      multiline={true}
```

Comment:
Will this change look & fell of all other widget dialogs that use this field?

Reply:
good point. it could be custom option just for text widget